README.md

```diff
@@ -17,6 +17,7 @@ The bot is used to manage and accompany the Discord server:
 ## Dependencies
 
 * [Docker Compose V3.8+](https://docs.docker.com/compose/install/)
+* [SQLite 3](https://www.sqlite.org/)
```

Comment:
The dev doesn't have to install SQLite though, why put this in the dependencies section?

Reply:
Discussed in meeting - we could use Docker for it but it's a headache SQLite is just needed for a dev script that's run once. Pulling an Alpine image, installed SQLite in there, running a single command then nuking it feels like overkill, devs can just install SQLite through their preferred package manager.